src/main/java/at/refugeesCode/kitchenchefbackend/endpoint/CookEndpoint.java

```diff
@@ -19,6 +19,7 @@
 
     private MealRepository mealRepository;
     private UserRepository userRepository;
+    private Meal meal;
```

Comment:
you have Created the Meal but you did added to the constructor, and you didn't make it autowird, so this won't work,
either you added to the constructor, which is the best practice or just remove it because we didn't use it,
we used the mealrepository